code/modules/mob/living/simple_animal/hostile/commanded/commanded.dm

```diff
@@ -22,7 +22,7 @@
 		command_buffer.Add(lowertext(html_decode(message)))
 	return 0
 
-/mob/living/simple_animal/hostile/commanded/Life()
+/mob/living/simple_animal/hostile/commanded/think()
 	while(command_buffer.len > 0)
```

Comment:
Side effect of this might be that commands get queued up while the mob is unconscious or dead.

Reply:
Hack but. Put this in there?
```DM
if (stat != ALIVE && command_buffer.len)
	command_buffer.Cut()
```